src/CitizenProject/Voter/CreateCitizenProjectVoter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace AppBundle\CitizenProject\Voter;
+
+use AppBundle\CitizenProject\CitizenProjectManager;
+use AppBundle\CitizenProject\CitizenProjectPermissions;
+use AppBundle\Entity\Adherent;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+class CreateCitizenProjectVoter implements VoterInterface
```

Comment:
Let's use `AbstractCitizenProjectVoter` no ?

Reply:
No because the `AbstractCitizenProjectVoter` uses a `CitizenProject`  subject, with the `CreateCitizenProjectVoter` it's not needed.